lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -407,9 +407,12 @@ dictionary PrepareReceiveRequest {
 };
 
 dictionary PrepareReceiveResponse {
-    u64? payer_amount_sat;
     PaymentMethod payment_method;
     u64 fees_sat;
+    u64? payer_amount_sat;
+    u64? zero_amount_min_payer_amount_sat;
+    u64? zero_amount_max_payer_amount_sat;
+    f64? zero_amount_service_feerate;
```

Comment:
The prefix zero_amount is a bit confusing IMO.
Why do we need this prefix? Why not showing these values on all cases?
min/max and service fees seem reasonable to be shown also on cases where amount is set.

Reply:
The only weird case is when `PrepareReceiveResponse.payment_method` is `LiquidAddress`. Then min/max don't mean much.

Should I then set them to `None` in that case?

Reply:
Yes I think that makes sense.

Reply:
Done: 1e18406e28ae4f16adff8cb4f9d525ce0a6aff55